config/initializers/lograge.rb

```diff
@@ -2,14 +2,17 @@
 
 Rails.application.configure do
   config.lograge.enabled = true
+  config.lograge.base_controller_class = ['ActionController::API', 'ActionController::Base']
   config.lograge.custom_payload do |controller|
     request = controller.request
     {
+      current_user_id: controller.current_user&.id,
```

Comment:
log `current_user` if logged in. it doesn't work nicely for `current_admin_user` when accessing API endpoints (can investigate further if necessary)